projects/fsastorefrontlib/src/core/product-assignment/facade/product-assignment.service.ts

```diff
@@ -63,6 +69,16 @@ export class ProductAssignmentService {
       .unsubscribe();
   }
 
+  isUserAdminOfUnit(unitId: string): Observable<boolean> {
+    return this.userService.get().pipe(
```

Comment:
Could we check this in HTML of the components themselves? Maybe no need for this additional method in service then?

Reply:
After discussion, we decided to keep it as is because the logic for checking if a user is the admin of unit will probably be needed for ProductAssignmnent guards.